src/dialog/reimu.c

```diff
@@ -15,21 +15,25 @@
 #define RIGHT_BASE cirno
 
 DIALOG_SCRIPT(reimu_stage1_pre_boss) {
-	LEFT("FIXME: Removed due to issue #179");
-	RIGHT("FIXME: Removed due to issue #179");
-	LEFT_FACE(unamused);
-	LEFT("FIXME: Removed due to issue #179");
-	RIGHT_FACE(angry);
-	RIGHT("FIXME: Removed due to issue #179");
-	RIGHT("FIXME: Removed due to issue #179");
+        LEFT_FACE(puzzled);
+        LEFT("Unseasonable snow? I wonder if I have to pay that 'Secret God' a visit again...");
+        RIGHT_FACE(normal);
+        RIGHT("'Secret God'? What, are you *that* impressed by invincible snow magic?!");
+        LEFT_FACE(sigh);
+        LEFT("Right, of course, I completely forgot about you. I should've known.");
+        RIGHT_FACE(angry);
+        RIGHT("Known better than to intrude on my secret base!!");
+        LEFT("Well, if you're getting excited, then I guess there really is something going on. I'd ask if you'd just let me through, but...");
+        RIGHT_FACE(normal);
+        RIGHT("No way! Prepare to get frozen!");
 }
 
 DIALOG_SCRIPT(reimu_stage1_post_boss) {
-	LEFT_FACE(happy);
-	LEFT("FIXME: Removed due to issue #179");
-	// RIGHT("FIXME: Removed due to issue #179");
-	LEFT_FACE(normal);
-	LEFT("FIXME: Removed due to issue #179");
+        LEFT_FACE(amused);
```

Comment:
Pretty sure this face doesn't exist ([here are all of them](https://github.com/taisei-project/taisei/tree/master/atlas/portraits/dialog)).

Doesn't really matter at this stage though; it's fine to completely ignore the faces for now. We can add them later, and I can probably ask @afensorm to draw extra expressions if we end up needing any, too.

Reply:
Changed that one, yep.